src/index.js

```diff
@@ -1,4 +1,32 @@
-import './styles/main.sass';
-import laughing from './assets/laughing.svg';
-const laughImg = document.getElementById('laughImg');
-laughImg.src = laughing;
+import './style.css';
+import arrow from './assets/arrow.svg';
+
+const tasks = [
+  { completed: false, description: 'Add tickbox', index: 4 },
+  { completed: false, description: 'Do laundry', index: 2 },
+  { completed: false, description: 'Take out trash', index: 3 },
+  { completed: false, description: 'Make dinner', index: 1 },
+];
+
+const todoListContainer = document.getElementById('todo');
+
+document.getElementById('addList');
+const arrowIcon = new Image();
+arrowIcon.src = arrow;
+document.getElementById('addList').appendChild(arrowIcon);
+
+function renderTodoList() {
+  todoListContainer.innerHTML = '';
+  tasks.sort((a, b) => a.index - b.index);
+  tasks.forEach((task) => {
+    const listItem = document.createElement('div');
+    listItem.className = 'todo-item';
+    listItem.innerHTML = `
+      <div class="checkbox"><i class="fa-regular fa-square"></i></div>
```

Comment:
As previously requested, please remove the style code from the UI, it will make for a bad UX

<img width="950" alt="image" src="https://github.com/dvcorreia1984/my-to-do-list/assets/84930355/13859fac-1362-4f2f-9bcb-5ba1f6fb2cb0">

Reply:
Hi Duane, I cant reproduce this error on my side. Can you kindly tell me how you opened this file on your side so I can reproduce the error on my side?

Reply:
Hi @DuaneDave, can you please tell me how you opened this file so I can reproduce the error please.

Reply:
I think I solved the problem now.